experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card class="statuses-details">
```

Comment:
This file is missing a license header; see other template HTML files for a comment at the top that you can copy.